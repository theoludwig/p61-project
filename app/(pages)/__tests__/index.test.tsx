import renderer from "react-test-renderer"

import HomePage from "@/app/(pages)/index"

describe("<HomePage />", () => {
  beforeEach(async () => {
    jest.clearAllMocks()
  })

  it("renders correctly", () => {
    const tree = renderer.create(<HomePage />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
