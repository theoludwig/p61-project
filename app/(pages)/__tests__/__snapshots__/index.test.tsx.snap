// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomePage /> renders correctly 1`] = `
<RNCSafeAreaView
  edges={
    {
      "bottom": "additive",
      "left": "additive",
      "right": "additive",
      "top": "additive",
    }
  }
  style={
    {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text
    style={
      {
        "color": "#006CFF",
        "fontFamily": "Georama",
        "fontSize": 36,
        "marginVertical": 20,
      }
    }
  >
    P61 Project
  </Text>
</RNCSafeAreaView>
`;
